embeddings.go

```diff
@@ -157,6 +157,9 @@ type EmbeddingRequest struct {
 	Model          EmbeddingModel          `json:"model"`
 	User           string                  `json:"user"`
 	EncodingFormat EmbeddingEncodingFormat `json:"encoding_format,omitempty"`
+	// Dimensions The number of dimensions the resulting output embeddings should have.
+	// Only supported in text-embedding-3 and later models.
+	Dimensions *int `json:"dimensions,omitempty"`
```

Comment:
Thank you for the PR!

Wouldn't `int` work just as well here? `0` would be ignored and all other values be included

Reply:
Thanks you sugguestion! I chang it type to `int`